iscdhcp/files/dhcpd.hosts

```diff
@@ -12,7 +12,9 @@
 host {{ h.name }} {
   hardware ethernet {{ h.mac }};
   fixed-address {{ h.ipaddr }};
+  {%- if h.get('insert-ddns-hostname', True) %} 
```

Comment:
please change to from True to False, so we keep default behaviour of the config. :)